install/tests/Makefile

```diff
@@ -528,10 +539,29 @@ destroy-azure:
 	ls ${KUBECONFIG} && terraform destroy -target=module.certmanager -var kubeconfig=${KUBECONFIG} --auto-approve || echo "No kubeconfig file"
 	terraform destroy -target=module.aks -var kubeconfig=${KUBECONFIG} --auto-approve
 
+restore-cert:
```

Comment:
Looks like you are duplicating existing actions in `create-secret` target https://github.com/gitpod-io/gitpod/pull/13817/files#diff-bd7b7f3a397df93111a6b6e2a4284b01bd57e04474bc9f91f1b02ee1960399a2L365 :thinking:  Any reason for this?

Reply:
🤦‍♂️ in my defense, reading code is hard 😅

Reply:
I don't blame you. And not to mention,  `Makefile`s 😿